treeherder/push_health/push_health.py

```diff
@@ -22,13 +23,13 @@
 ]
 
 
-def get_intermittent_history(prior_day, days, option_map):
+def get_history(failure_classification_id, prior_day, days, option_map):
     start_date = datetime.datetime.now() - datetime.timedelta(days=days)
     repos = Repository.objects.filter(name__in=['mozilla-inbound', 'autoland', 'mozilla-central'])
     failure_lines = FailureLine.objects.filter(
         job_log__job__result='testfailed',
         job_log__job__tier=1,
-        job_log__job__failure_classification_id=4,
+        job_log__job__failure_classification_id=failure_classification_id,
         job_log__job__push__repository__in=repos,
```

Comment:
I'm curious if this part is slow (since you mentioned it in your comments). I'm guessing you are querying for repos first in line 28 because querying by `repository__name__in` was slow? (It was for me when working on an API for IFV and it seems like it might be a MYSQL issue). I'm curious if filtering by `repository_id__in=repos_ids` affects the query time versus querying by the entire object?

Reply:
It does seem a bit faster to do it with the ids.  Though something like 3.6s, rather than 3.8.  But I'll take it.  :)  Good suggestion.  I also just use the hard-coded ids, rather than having to query for the repo ids each time.  Since these come from fixtures, I figure that's ok.

Reply:
Just remembered I have the id's hard-coded in utils already: https://github.com/mozilla/treeherder/blob/master/treeherder/webapp/api/utils.py#L12

Reply:
ahh, cool.  fixed to use that.